interface/weno/formularyInsert.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
What is the difference between 2 above files drugPaidInsert.php and drug_paid.php ?